homeassistant/config_entries.py

```diff
@@ -685,6 +681,18 @@ def async_start_reauth(
             )
         )
 
+    @callback
+    def async_get_active_flows(
+        self, hass: HomeAssistant, sources: set[str]
```

Comment:
`set[str]` is obviously a little loose, but I figure that if someone passes an invalid source, they just won't get results back. Reasonable?

Reply:
Making the source a `StrEnum` would be a great long term solution for a future PR.

`set[str]` is the best we can do for now.